<?xml version="1.0" encoding="ISO-8859-15"?>
<launch>

  <!-- Load Arguments -->
  <arg name="slamMode" default="0"/> 

  <!-- Load parameters -->
  <rosparam command="load" file="$(find cans)/config/unrealNurbslamParameters.yaml" />
  <!-- <rosparam command="load" file="$(find cans)/config/nurbslamParameters.yaml" /> -->

  <!-- Run node -->
  <node pkg="cans" type="nurbslam_node" name="nurbslam_node" args="$(arg slamMode)" output="screen"/>
  
</launch>
